xoa/data_source_hosts_test.go

```diff
@@ -76,6 +76,8 @@ func getCompositeAggregateTestFunc(resourceName, sortBy, sortOrder string) resou
 	return resource.ComposeAggregateTestCheckFunc(
 		testAccCheckXenorchestraDataSourceHosts(resourceName),
 		resource.TestCheckResourceAttrSet(resourceName, "id"),
+		resource.TestCheckResourceAttrSet(resourceName, "hosts.0.cpus.sockets"),
+		resource.TestCheckResourceAttrSet(resourceName, "hosts.0.memory"),
```

Comment:
Can you add similar assertions to the `xoa/data_source_host_test.go` file as well?